jekyll/_cci2/high-uid-error.md

```diff
@@ -68,4 +84,14 @@ $ ls -ln file-with-high-id
 -rw-r--r-- 1 1000000 1000000 0 Jul  9 03:05 file-with-high-id
 ```
 
-Once you find which file has a high UID/GID, make sure to change the ownership of the file and re-create the image.
+After locating the offending file,
+change its ownership
+and re-create the image.
+
+**Note:**
+It is possible for an invalid file to be generated and removed
+while a container is building.
+In this case,
+you may have to modify the `RUN` step in the Dockerfile itself.
+Adding `&& chown -R root:root /root` to the problem step
```

Comment:
Is it reasonable to assume that users can identify which step to add this command?

Reply:
This gets into some philosophical territory for me, actually: how deep do we want to go to explain things? Is it _our fault_ that this bug even exists? If it is, then I'd say we need to provide a complete workaround.